tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -4359,6 +4359,7 @@ declare namespace ts {
     function setSyntheticTrailingComments<T extends Node>(node: T, comments: SynthesizedComment[] | undefined): T;
     function addSyntheticTrailingComment<T extends Node>(node: T, kind: SyntaxKind.SingleLineCommentTrivia | SyntaxKind.MultiLineCommentTrivia, text: string, hasTrailingNewLine?: boolean): T;
     function moveSyntheticComments<T extends Node>(node: T, original: Node): T;
+    function ignoreSourceNewlines<T extends Node>(node: T): T;
```

Comment:
As I said above, I'm not sure this needs to be part of the public API at this time.